### PR TITLE
Fixes winter wonderland and emergency medical

### LINKED ISSUE #18717: Winter wonderland at some point got bugged
image

### LINKED ISSUE #18783: Medical holodeck floor is busted
image

### PR DESCRIPTION
**Pull request by pudl**
no u

There's a wooden chair in ww now to respect the legend of the chair wonderland.
I've never seen the emergency medical and I have no reference so I'm just guessing on how it looked.

Fixes #18717 
Fixes #18783

### CODE REVIEW

File: _maps/map_files/generic/z2.dmm [hunk 1]
```diff
@@ -277,16 +277,18 @@
...
 	},
 /area/holodeck/rec_center/medical)
 "aZ" = (
 /obj/structure/table/glass,
 /obj/item/weapon/hemostat,
-/turf/open/floor/holofloor{
-	icon_state = "warnwhite";
+/turf/open/floor/plasteel/warnwhite{
+	tag = "icon-white_warn (NORTH)";
+	icon_state = "white_warn";
```
Comment (xxalpha) [inline _maps/map_files/generic/z2.dmm]: How is this here?

Comment (pudl) [inline _maps/map_files/generic/z2.dmm]: Im stupid thats how

File: _maps/map_files/generic/z2.dmm [hunk 2]
```diff
@@ -297,23 +301,29 @@
...
 	},
 /area/holodeck/rec_center/medical)
 "bc" = (
 /obj/structure/table/glass,
 /obj/item/weapon/cautery,
 /turf/open/floor/holofloor{
-	icon_state = "warnwhite";
+	tag = "icon-white_warn (NORTHEAST)";
+	icon = 'icons/turf/floors/warning.dmi';
+	icon_state = "white_warn";
```
Comment (xxalpha) [inline _maps/map_files/generic/z2.dmm]: It's still around. Did you edit this or what happened? WJ had this in his lavaland edit as well.

Comment (pudl) [inline _maps/map_files/generic/z2.dmm]: What are you talking about? What's wrong with it?

Comment (xxalpha) [inline _maps/map_files/generic/z2.dmm]: Wait, these are holofloors, nevermind.

**Comment by Fox-McCloud** [comment]
inb4 people use holo medical to perform surgeries and skip out on medbay entirely

**Comment by ChangelingRain** [comment]
You are a liiiiiittle late, these are already in.

**Comment by Fox-McCloud** [comment]
Yeah, I know =p

**Pull request merged**